packages/rocketchat-i18n/i18n/de.i18n.json

```diff
@@ -1662,6 +1664,10 @@
   "Report_sent": "Bericht gesendet",
   "Report_this_message_question_mark": "Diese Nachricht melden?",
   "Reporting": "Berichtswesen",
+  "Request": "Anfrage",
```

Comment:
This should be part of the assistify-file

Reply:
These changes must be contributed to RC.